tests/test_switch_keep_alive.py

```diff
@@ -210,6 +210,7 @@ async def test_switch_keep_alive(self, common_mocks: CommonMocks):
             common_mocks,
             [call("switch", SERVICE_TURN_ON, {"entity_id": "switch.mock_switch"})],
         )
+        common_mocks.mock_is_state.return_value = True
```

Comment:
Because the callback calls are now conditional on `self.is_device_active`, and the Home Assistant state is mocked.